### ISSUE TITLE
Reskin: Replace Plus icon(Indicating to add column) in create datatable.

### ISSUE THREAD

**Issue by gkrishnan724** [openMF/community-app#2153]
I feel the `+` icon in the add column section for the create datatable is not informative enough. In my case, I forgot to add the column and I pressed the submit button which gave me an error. Then I realised we need to add columns by using the `+` icon. I would suggest replacing the plus icon with a button like `+Add column` which would be more informative

![btnprob](https://cloud.githubusercontent.com/assets/21004514/24358974/757316a8-1320-11e7-927d-0d56ccee7d54.png)

**Comment by mbj36** [comment]
Design should be same through out the community app, Changing the + button in one place would go against the design pattern in community app but still if you wish to change, then you must gather all the cases where + button is present and create a check box here and then you can start changing them

**Comment by gkrishnan724** [comment]
@mbj36 True, I totally agree with the fact that design should be consistent in the community-app. However, I feel that the '+' icon here is not proper because, In the community-app we have been using + for something like `showing more fields` (consider the example of creating a new client where we add address or  debit and credit fields in add journal entry). However, In this case plus icon acts more like a submit column button. A more appropriate use of + here would be to show more column fields to add in the datatable(This is what I had interpreted it as) not to submit the entered data in the field. what do you think?

**Comment by santoshmath** [comment]
@gkrishnan724 , @mbj36 , it could be considered as 'improvement' issue. In order to maintain consistency as Mohit  said, first we have to list all the pages in Mifos application where such scenarios exist.

**Comment by Rupeshiya** [comment]
@gkrishnan724  should I replace `+` with Add Column button ? I would like to fix this .
